external/Android.mk

```diff
@@ -17,6 +17,7 @@ sqlcipher/sqlite3.c:
 	make -C sqlcipher sqlite3.c
 
 copy-libs-hack: build-local-hack
+	cd ${CURDIR}
```

Comment:
Without that line, the target fails when run after build-local-hack (because we are in the wrong directory)